test/replcompletions.jl

```diff
@@ -663,59 +666,57 @@ c, r, res = test_complete(s)
 @test isempty(c)
 
 # test Dicts
-test_dict = Dict("abc"=>1, "abcd"=>10, :bar=>2, :bar2=>9, Base=>3, contains=>4, `ls`=>5,
-      66=>7, 67=>8, ("q",3)=>11, "α"=>12, :α=>13)
-s="test_dict[\"ab"
+s="CompletionFoo.test_dict[\"ab"
```

Comment:
module qualified vs not qualified by a module. it seems like you're no longer testing the unqualified version?